scripts/run_attack_args_helper.py

```diff
@@ -34,21 +34,28 @@
     'lstm-imdb':                'textattack.models.classification.lstm.LSTMForIMDBSentimentClassification',
     'lstm-mr':                  'textattack.models.classification.lstm.LSTMForMRSentimentClassification',
     'lstm-yelp-sentiment':      'textattack.models.classification.lstm.LSTMForYelpSentimentClassification',
-    
     #
     # Textual entailment models
     #
-    
     # BERT models
     'bert-mnli':                'textattack.models.entailment.bert.BERTForMNLI',
     'bert-snli':                'textattack.models.entailment.bert.BERTForSNLI',
+    #
+    # Translation models
+    #
+    't5-en2fr':                 'textattack.models.translation.t5.T5EnglishToFrench',
+    't5-en2de':                 'textattack.models.translation.t5.T5EnglishToGerman',
+    't5-en2ro':                 'textattack.models.translation.t5.T5EnglishToRomanian',
+    #
+    # Summarization models
+    #
+    't5-summ':                  'textattack.models.summarization.T5Summarization',
```

Comment:
So we have a summarization model but no summarization datasets yet right

Reply:
yep (see PR description)